github/repos.go

```diff
@@ -2438,3 +2438,96 @@ func (s *RepositoriesService) IsPrivateReportingEnabled(ctx context.Context, own
 	resp, err := s.client.Do(ctx, req, privateReporting)
 	return privateReporting.Enabled, resp, err
 }
+
+// RepositoryActivityOptions specifies the optional parameters to the
+// RepositoriesService.ListRespositoryActivities method.
+type RepositoryActivityOptions struct {
+	// The direction to sort the results
```

Comment:
Let's extend the description:
```suggestion
	// The direction to sort the results by. Can be one of: `asc`, `desc`. Default: `desc`. 
```

This can be found at GitHub docs:

<img width="434" height="195" alt="image" src="https://github.com/user-attachments/assets/dc54ba0d-d407-4db4-b9fa-0402d0f2c636" />

Reply:
The same comment applies to other fields.

Reply:
Taken care